tests/pyoozie/test_client.py

```diff
@@ -188,6 +212,7 @@ def test_request(self, api):
             m.get('http://localhost:11000/oozie/v2/endpoint', text='{"result": "pass"}')
             result = api._request('GET', 'endpoint', None, None)
             assert result['result'] == 'pass'
+            
```

Comment:
🔥

Reply:
I've been too spoiled by pre push hooks not letting me push silly linting errors